samples/apps/copilot-chat-app/webapp/src/redux/features/conversations/conversationsSlice.ts

```diff
@@ -32,16 +35,34 @@ export const conversationsSlice: Slice<ConversationsState> = createSlice({
             state.conversations = { [newId]: action.payload, ...state.conversations };
             state.selectedId = newId;
         },
-        updateConversation: (
+        addUserToConversation: (state: ConversationsState, action: PayloadAction<{ user: IChatUser, chatId: string }>) => {
+            const { user, chatId } = action.payload;
+            state.conversations[chatId].users.push(user);
+        },
+        /*
+        * updateConversationFromUser() and updateConversationFromServer() both update the conversations state.
+        * However they are for different purposes. The former action is for updating the conversation from the
```

Comment:
You mention the former action here (updateConversationFromUser) and then updateConversationFromUser in line 47, are both lines meant to describe the same action?